staticjinja/staticjinja.py

```diff
@@ -59,18 +81,20 @@ def __init__(self,
                  outpath,
                  encoding,
                  logger,
-                 contexts=None,
+                 contexts=[],
```

Comment:
It's usually a subtle bug to default to a mutable object any function, so we'll want to change this back. See http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments